osf/models/user_message.py

```diff
@@ -87,7 +87,7 @@ def send_institution_request(self) -> None:
         send_mail(
             mail=MessageTypes.get_template(self.message_type),
             to_addr=self.recipient.username,
```

Comment:
I think, in the case of BCC'ing the sender, the recipient also needs to be BCC'd. There would be no to_address in that case. Without the BCC, however, it makes sense to have the recipient be in the `to_addr`.

Reply:
This is just a wrapper for the the two different email utilities that implement the BBC in different ways, so it's hard to say which behavior is actually "the best". 

But anyway, if there's no `to_addr` and just `bbc`-ing, yes that can just BBC I just have to that meld with the two utility functions, seems reasonable from a developer experience perceptive to adapt that behavior to SendGrid and simple SMTP, but it's just making the `send_email` wrapper around SMTP and Sendgrid better if I understand correctly.

Reply:
Yeah, I'm not super-concerned where it's changed, just that all recipients of the email are bcc'd if the bcc option is chosen.

Reply:
Yeah I dealt with that here: 
```
recipients = [to_addr] + (bcc_addr or [])
```
https://github.com/CenterForOpenScience/osf.io/pull/10841/files#diff-78b71751c095a6eb9a92be212c4acf17f402fc34266030b5440596cf2a57d243R112 sorry probably got lost with the clean-up, but that's new. Sendgrid has it's own way of handling it.

Reply:
Cool cool